app/models/user.rb

```diff
@@ -15,4 +16,26 @@ def User.digest(string)
                                                   BCrypt::Engine.cost
     BCrypt::Password.create(string, cost: cost)
   end
+
+  # ランダムなトークンを返す
+  def User.new_token
+    SecureRandom.urlsafe_base64
+  end
+
+  # 永続セッションのためにユーザーをデータベースに記憶する
+  def remember
+    self.remember_token = User.new_token
+    update_attribute(:remember_digest, User.digest(remember_token))
```

Comment:
```suggestion
    update(remember_digest: User.digest(remember_token))
```
FYI
これ実は非推奨なメソッドで、6.1系以降から削除されています。（validation が効かないので）
通常`update`もしくは`update!`を用い、例外を出したいか否かで使い分けます。